Gemfile

```diff
@@ -49,3 +49,16 @@ gem 'rubocop', require: false
 gem 'active_model_serializers', '~> 0.10.0'
 
 gem 'enumify'
+
+group :development, :test do
+  gem 'rspec-rails', '~> 3.8'
+  gem 'shoulda-matchers', '~> 4.0'
+end
+
+group :test do
```

Comment:
what's the difference between this group and the previous one? seems like both relate to tests.